girder/molecules/server/experiment.py

```diff
@@ -80,15 +81,27 @@ def create(self, body):
     def find_experiment(self, params):
         user = getCurrentUser()
 
-        query = { }
-        if 'molecularFormula' in params:
-            query['molecularFormula'] = params['molecularFormula']
+        if 'source' in params.keys():
```

Comment:
`'source' in params` is enough. Actually where is `source` used?

Reply:
I put it there because the function can be extended to use later other sources different from NIST.

Reply:
I plan to let users decide the `source` to query the spectra.